### PR TITLE
Note that algorithmic facial matches are considered biometric matches

### LINKED ISSUE #392: Recommend ID document verification in accord with SP800-63A IAL3
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 880-885
 
**Comment (Include rationale for comment)**: "If the biometric verification decision is negative, or if no biometric data records are available, the cardholder SHALL provide two identity source documents (as specified in Section 2.7), and an attending operator SHALL inspect these and compare the cardholder with the electronic facial image retrieved from the enrollment data record and the photograph printed on the new PIV Card."
 
**Suggested Change**: §2.7 requirements, those of 842-845, and here, should line up.  Recommend ID document verification in accord with SP800-63A IAL3, not just specifying the document types to use.
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### PR DESCRIPTION
**Pull request by jimfenton**
Fixes #392

### CODE REVIEW

**Comment by gfiumara** [comment]
Is this necessary? If we have to add this footnote here for clarity, I'd suggest that we are unclear about what facial recognition in the remainder of the document and that we should edit it more broadly (if we haven't already: see #639).

**Comment by hferraio** [comment]
I am not sure we address comment 392 or 402, 405 396.   The comment, to me, is about when biometric could NOT be matched and have to resort to staff to manually compares cardholder's face to digital representation of face biometrics on-record or card.  This can happen (maybe a person with Parkinson Disease - automated matching is difficult?).  We sometimes have slightly different language/method with processes for the manual comparison.  That is what we should looked into, I believe.

File: _FIPS201/requirements.md [hunk 1]
```diff
@@ -333,7 +333,7 @@ of the federal department or agency.
 + During the issuance process, the issuer **SHALL** verify that the individual to whom the PIV Card is to be
     issued is the same as the intended applicant/recipient as approved by the appropriate authority.
     Before the PIV Card is provided to the applicant, the issuer **SHALL** perform a one-to-one comparison of the
-    applicant against biometric data records available on the PIV Card or in the PIV enrollment record.
+    applicant against biometric data records available on the PIV Card or in the PIV enrollment record[^facial].
```
Comment (jricher) [inline _FIPS201/requirements.md:336]:
```suggestion
    applicant against biometric data records available on the PIV Card or in the PIV enrollment record.[^facial]
```

File: _FIPS201/requirements.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ It **SHALL** be printed on the PIV Card according to [Section 4.1.4.1](frontend.
 The image **MAY** be used for cardholder authentication (BIO or BIO-A) as described in [Section 6.2.1](authentication.md#s-6-2-1).
 It **MAY** be retrieved and displayed on guard workstations to augment 
 other authentication processes from [Section 6.2](authentication.md#s-6-2).
-The electronic facial image is an additional means of authentication during PIV issuance and maintenance processes.
+The electronic facial image is an additional means of authentication during PIV issuance and maintenance processes when used with an automated facial recognition algorithm.
```
Comment (gfiumara) [inline _FIPS201/requirements.md:139]:
```suggestion
The electronic facial image is an additional means of authentication during PIV issuance and maintenance processes when used with an automated facial comparison algorithm.
```

**Pull request merged**